up42/viztools.py

```diff
@@ -2,617 +2,6 @@
 Visualization tools available in various objects
```

Comment:
Why do we need to keep the module itself?

Reply:
some other classes (for example [see this](https://github.com/up42/up42-py/blob/4c35515cab21e50ff8b8a233454e0f0036289508/up42/job.py#L18)), depend on class `VizTools`.
And I wanted to tackle it in multiple PRs.